pokemongo_bot/cell_workers/transfer_pokemon.py

```diff
@@ -5,7 +5,13 @@
 from pokemongo_bot.cell_workers.base_task import BaseTask
 
 class TransferPokemon(BaseTask):
+    def initialize(self):
```

Comment:
I think we need a smarter way to speed up this task, I'm definitely a :-1: on checking the tick. I don't even like that we have a flag for the first tick.

Another approach here would be for the task to have a list of the pokemon it saw on the last tick, and then on the next tick if the pokemon have changed, run the transfer check on only the relevant ones.

I think this should be undone in this PR and done properly in a standalone PR.